packages/esm-home-app/src/homepage-header/homepage-header.scss

```
@@ -0,0 +1,85 @@
+@use '@carbon/colors';
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
```

Nitpicking, but it's possible to use these shorter imports here:

```suggestion
@use '@carbon/layout';
@use '@carbon/type';
```

The caveat with `layout` is that you'd have to use the `layout` namespace instead of the `spacing` one. So, `layout.$spacing-12` instead of `spacing.$spacing-12`.